### PR TITLE
New Space Ruin: The All-American Diner!

### PR DESCRIPTION
**Pull request by Singul0**
## About The Pull Request
Adds a new space ruin based off the aesthethics of a typical 1950's american diner. Image below:

## Why It's Good For The Game
There is a distinct lack of food themed space ruins. icebox has the pizza restaurant, lavaland has the pizza party. it's about time space has a food themed ruin too!
## Changelog
:cl:
add: Due to an influx of syndicate activities near the station, A number of businesses in the nearby sector of space have been closed down. One of which being a themed old-style american diner. Can you find it? 
/:cl:

### CODE REVIEW

**Comment by Singul0** [comment]
> You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

Once this #74369 gets merged it's actually 0.

**Comment by Singul0** [comment]
image
How it looks like in-game.

**Comment by tralezab** [comment]
reminds me of the spacebar

File: code/modules/mapfluff/ruins/spaceruin_code/allamericandiner.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//
+/obj/item/paper/fluff/ruins/allamericandiner
+	name = "Notice of Mothballing"
+	default_raw_text = "Due to the recent increase in Syndicate raids, The subsequent decrease in customers and the inability of the SCC to defend this subsector against hostile threats. Corporate has decided to mothball this station and wait for the whole situation to calm down. Pack up your stuff, Clean the station the best you could. A shuttle will pick you all up in 14 cycles"
```
Comment (tralezab) [inline code/modules/mapfluff/ruins/spaceruin_code/allamericandiner.dm:4]:
```suggestion
	default_raw_text = "Due to the recent increase in Syndicate raids, The subsequent decrease in customers and the inability of the SCC to defend this subsector against hostile threats. Corporate has decided to mothball this station and wait for the whole situation to calm down. Pack up your stuff, Clean the station the best you can. A shuttle will pick you all up in 14 cycles."
```

**Comment by AdipemDragon** [comment]
on the screenshot alone, that jukebox kind of makes that one seating area inaccessible without hopping the table (yes i know it doesn't really matter because it's a space ruin but)

**Comment by Singul0** [comment]
> on the screenshot alone, that jukebox kind of makes that one seating area inaccessible without hopping the table (yes i know it doesn't really matter because it's a space ruin but)

I was debating on moving it replacing the soda machine but the floor became way too garish to look at without something to break it up in the bottom left of the room.

**Comment by Singul0** [comment]
Alright map is done. Just need a maptainer to review it.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**